SketchUp/Sketchup/Model.rb

```diff
@@ -75,15 +75,16 @@ class Sketchup::Model
   def abort_operation
   end
 
-  # The active_entities method is used to retrieve an Entities object containing
+  # The active_entities method is used to retrieve an {Entities} object containing
   # all of the entities in the active model, component, or group (if you are
   # within a group or component edit session.)
   #
   # @example
   #   model = Sketchup.active_model
   #   entities = model.active_entities
   #
-  # @return [Sketchup::Entities] entities - the {Sketchup::Entities} of the current editing context, or Model{Model#entities} if none.
+  # @return [Entities] The {Entities} of the current editing context,
```

Comment:
Don't we need to use fully qualified identifiers? `Sketchup::Entities` instead of `Entities`? Or is just `Entities` working for the stubs? (In which case I need to verify it works when generating from our C++ source)